cppcache/integration/test/CachingProxyTest.cpp

```diff
@@ -51,86 +53,67 @@ class CachingProxyTest : public ::testing::Test {
 
   ~CachingProxyTest() override = default;
 
-  void SetUp() override {}
+  void SetUp() override {
+    cache.getPoolManager()
+        .createFactory()
+        .addLocator("localhost", cluster.getLocatorPort())
+        .create("pool");
+
+    region = cache.createRegionFactory(RegionShortcut::CACHING_PROXY)
+                 .setPoolName("pool")
+                 .create("region");
+  }
 
   void TearDown() override {}
 
   Cluster cluster = Cluster{LocatorCount{1}, ServerCount{1}};
+  Cache cache = CacheFactory().create();
   std::string key = std::string("scharles");
   std::string value = std::string("Sylvia Charles");
+  std::shared_ptr<Region> region;
 };
 
 TEST_F(CachingProxyTest, LocalRemoveAfterLocalInvalidate) {
-  auto cache = CacheFactory().create();
-
-  cache.getPoolManager()
-      .createFactory()
-      .addLocator("localhost", cluster.getLocatorPort())
-      .create("pool");
-
-  auto region = cache.createRegionFactory(RegionShortcut::CACHING_PROXY)
-                    .setPoolName("pool")
-                    .create("region");
-
   region->put(key, value);
 
   auto user = region->get(key);
 
   region->localInvalidate(key);
 
-  bool resultLocalRemove = region->localRemove(key, value);
+  auto resultLocalRemove = region->localRemove(key, value);
   ASSERT_FALSE(resultLocalRemove);
 
   resultLocalRemove = region->localRemove(
-      key, static_cast<std::shared_ptr<Cacheable>>(nullptr));
+      // key, static_cast<std::shared_ptr<Cacheable>>(nullptr));
```

Comment:
Delete commented out code and format. Repeated in a few other places.

Reply:
Good catch. Thorough, that jbarrett is.